src/main/java/com/flipkart/databuilderframework/engine/DataBuilderExecutionListener.java

```diff
@@ -31,10 +32,12 @@ void afterException(
             Map<String, Data> prevResponses,
             Throwable frameworkException) throws Exception;
 
+
     default void postProcessing(DataFlowInstance dataFlowInstance,
                                 DataDelta dataDelta,
                                 DataExecutionResponse response,
-                                Throwable frameworkException) throws Exception {
+                                Throwable frameworkException,
+                                DataBuilderContext dataBuilderContext) throws Exception {
```

Comment:
Instead of removing the older function, create this as a new function with call to older function as default implementation.

Reply:
changed it.